### PR TITLE
[py-mizani] added version 0.8.1

### PR DESCRIPTION
**Pull request by qwertos**
Depends on: #36499

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the sha256.

File: var/spack/repos/builtin/packages/py-mizani/package.py [hunk 1]
```diff
@@ -25,9 +26,12 @@ class PyMizani(PythonPackage):
     depends_on("python@3.8:", type=("build", "run"), when="@0.7.4:")
     depends_on("python@3.6:", type=("build", "run"), when="@0.6.0:")
 
+    depends_on("py-setuptools@45:", type="build", when="@0.8.1:")
     depends_on("py-setuptools@42:", type="build", when="@0.7.4:")
     depends_on("py-setuptools", type="build", when="@0.6.0:")
 
+    depends_on("py-setuptools-scm@6.2:", type="build", when="@0.8.1:")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-mizani/package.py:33]:
```suggestion
    depends_on("py-setuptools-scm@6.2:+toml", type="build", when="@0.8.1:")
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — DISMISSED** [review]

**Comment by qwertos** [comment]
I guess I shouldn't use github's rebase feature...

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Yep, GitHub's rebase feature is broken at the moment, @alalazo reported the same thing

**Comment by adamjstewart** [comment]
https://github.com/orgs/community/discussions/51948

**Pull request merged**